Projects/Features/SearchFeature/Sources/ViewControllers/BeforeSearchContentViewController.swift

```diff
@@ -48,6 +48,24 @@ public final class BeforeSearchContentViewController: BaseStoryboardReactorViewC
         reactor?.action.onNext(.viewDidLoad)
     }
 
+    override public func addView() {
+        super.addView()
+        self.view.addSubviews(collectionView, tableView)
+        configureDataSource()
+    }
+
+    override public func setLayout() {
+        super.setLayout()
+
+        tableView.snp.makeConstraints {
+            $0.horizontalEdges.verticalEdges.equalToSuperview()
+        }
+
+        collectionView.snp.makeConstraints {
+            $0.verticalEdges.horizontalEdges.equalToSuperview()
```

Comment:
```suggestion
            $0.edges.equalToSuperview()
```
이친구 그냥 요렇게 해도 되지 않나요

Reply:
수정했습니다.